weneige/orders/urls.py

```diff
@@ -0,0 +1,6 @@
+from django.urls import path
+from .views      import OrderView
+
+urlpatterns = [
+    path("/order", OrderView.as_view()),
```

Comment:
```suggestion
    path("", OrderView.as_view()),
```
아래와 같이 `/order`를 지우더라도 충분히 RESTful 할 것 같습니다!
```
#before
http -v GET localhost:8000/orders/order
http -v POST localhost:8000/orders/order  ...   ...  ...

#after
http -v GET localhost:8000/orders
http -v POST localhost:8000/orders  ...   ...  ...
```

Reply:
넵 알겠습니다.